src/Entity/Client.php

```diff
@@ -0,0 +1,354 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\ClientRepository;
+use DateTimeImmutable;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Ramsey\Uuid\UuidInterface;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+/**
+ * @ORM\Entity(repositoryClass=ClientRepository::class)
+ */
+class Client implements UserInterface
+{
+    /**
+     * @var int id
+     *
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer", name="bm_id")
+     */
+    private int $id;
```

Comment:
Codacy found an issue: [Avoid variables with short names like $id. Configured minimum length is 3.](https://app.codacy.com/gh/CarolineDirat/P7APIBileMo/pullRequest?prid=6297993)